### PR TITLE
feat(tianmu): rename uniform code issue11

### PR DESCRIPTION
**Pull request by hustjieke**
## Summary about this PR

Issue Number: close #issue_number_you_created

## Tests Check List

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog

- [ ] New Feature
- [ ] Bug Fix
- [x] Improvement
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

File: storage/tianmu/handler/ha_tianmu.h [hunk 1]
```diff
@@ -162,7 +162,7 @@ class ha_tianmu final : public handler {
```
Comment (lujiashun) [inline storage/tianmu/handler/ha_tianmu.h:162]: dupkey_pos of uint is a positive value  assigne to a negative value -1, better to assign it to UINT32_MAX (defined in stdint.h)

File: storage/tianmu/handler/ha_tianmu.h [hunk 2]
```diff
@@ -171,7 +171,7 @@ class ha_tianmu final : public handler {
   core::TableID tmp_table_;
   std::unique_ptr<core::CompiledQuery> cq_;
```
Comment (lujiashun) [inline storage/tianmu/handler/ha_tianmu.h]: some unique_ptr variables named with suffix  _ptr_, while some variables named without the suffix. Better to unform it.

File: storage/tianmu/handler/ha_tianmu.h (hunk 1 above)
Comment (hustjieke) [reply storage/tianmu/handler/ha_tianmu.h:162]: Fixed.

Comment (hustjieke) [reply storage/tianmu/handler/ha_tianmu.h:162]: Fixed.

**Review by lujiashun — APPROVED** [review]
LGTM

**Review by DandreChen — APPROVED** [review]
LGTM.

**Review by RingsC — APPROVED** [review]

**Pull request merged**